src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -252,6 +252,7 @@ EOF;
     }
 
     // Set up forwarding if it configured
+    $forward_conf = '';
```

Comment:
also safe string interpolation later. the real WTF is that it's embedded and written even if empty, which forces an unwanted newline. but why not... but delete else { $forward_conf = ""; } ;)